route.go

```diff
@@ -138,12 +138,17 @@ func cmdRouteRemove(args []string, home string) {
 	if err := json.Unmarshal(b, &existing); err != nil {
 		log.Fatalf("unmarshaling routes: %s", err)
 	}
-	_, ok := existing[name]
-	if !ok {
-		log.Fatalf("route %q not found; not making any changes.", name)
+
+	for _, name := range names {
+		_, ok := existing[name]
+		if !ok {
+			log.Fatalf("route %q not found; not making any changes.", name)
```

Comment:
a thought on wording. let's suppose the command was invoked like: `lyft route remove home office soccer`, and that the `office` route does not exist. 

The output would be:
```
lyft: removed route "home"
lyft: route "office" not found; not making any changes.
```

which might be confusing, because the two lines say conflicting things. my suggestion is to remove the log on line 149. then the output would be the single line:

```
lyft: route "office" not found; not making any changes.
```

which is accurate, since the command would exit and not persist the changes (i.e. `writeRoutes` below isn't run).

Reply:
another nice thing about this approach is that we would have the UNIX behavior of "don't print anything if things worked as expected".